examples/affine_transforms.py

```diff
@@ -25,7 +26,14 @@
     viewer.add_image(image, colormap='blue', opacity=.5, name='moving', affine=affine)
     viewer.add_points((corners_h @ affine.T)[:, :-1], size=0.5, opacity=.5, face_color=[0, 0, 0.8, 0.8], name='mv corners')
 
-    # Reset the view
-    viewer.reset_view()
+    # Note how the transformed corner points remain at the corners of the transformed image
+
+    # Now add the a regridded version of the image transformed with scipy.ndimage.affine_transform
```

Comment:
I think it's worth explaining *why* you are showing this comparison. Are you making the argument that it's better for people to use the napari affine transformation because the regridding/resampling causes some distortions in the way the data looks? I think that's a valid argument to make.

We should try and be a lot clearer about what take home message we are expecting users to understand from this example.

Reply:
I like that idea! We can do that when we write up a tutorial on basic usage of the affine

Reply:
...isn't *this* the tutorial on basic affine use?

Reply:
Not yet! Tutorials live in `napari/tutorials`. *Eventually*, we want to unify them. That hasn't happened yet though.

Reply:
Yeah, I was thinking one of our markdown files in the tutorials repo, but agreed I can't wait till we can unify them. It will change everything making development of tutorials much easier and more fun!!!